ci/cpu/build.sh

```diff
@@ -15,11 +15,7 @@ cd "$WORKSPACE"
 # Get latest tag and number of commits since tag
 export GIT_DESCRIBE_TAG=`git describe --abbrev=0 --tags`
 export GIT_DESCRIBE_NUMBER=`git rev-list ${GIT_DESCRIBE_TAG}..HEAD --count`
-
-# If nightly build, append current YYMMDD to version
-if [[ "$BUILD_MODE" = "branch" && "$SOURCE_BRANCH" = branch-* ]] ; then
-  export VERSION_SUFFIX=`date +%y%m%d`
-fi
+export RAPIDS_DATE_STRING=$(date +%y%m%d)
```

Comment:
Should this be `git show --no-patch --date=format:'%y%m%d' --format='%cd'` like in GHA: https://github.com/rapidsai/gha-tools/blob/main/tools/rapids-env-update#L23

Reply:
I will be submitting a PR to `gha-tools` to replace that line with some logic that gets the date from the corresponding GHA workflow run.

Since some of these repos aren't on GHAs yet, it's fine to just let them use `date` for now.